e2e/cypress/integration/scroll/fixed_width_spec.js

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @scroll
+
+describe('Scroll', () => {
+    let testTeam;
+    let testChannel;
+    let otherUser;
+
+    beforeEach(() => {
+        // # Create new team and new user and visit Town Square channel
+        cy.apiInitSetup().then(({team, channel}) => {
+            testTeam = team;
+            testChannel = channel;
+
+            cy.apiCreateUser().then(({user: user2}) => {
+                otherUser = user2;
+
+                cy.apiAddUserToTeam(testTeam.id, otherUser.id).then(() => {
+                    cy.apiAddUserToChannel(testChannel.id, otherUser.id);
+                });
+            });
+
+            cy.visit(`/${testTeam.name}/channels/${channel.name}`);
+        });
+    });
+
+    it('MM-T2368 Fixed Width setting should not scroll pop and display posts properly', () => {
+        // # Creating some post to verify scroll pop and Posts views
+        cy.postMessage('This is the first post');
+        Cypress._.times(18, (postIndex) => {
+            cy.postMessage(`p-${postIndex + 1}`);
+        });
+        cy.postMessage('This is the last post');
+
+        // # Switch the account settings for the test user to enable Fixed width center
+        cy.toAccountSettingsModal();
+        cy.get('#displayButton', {timeout: 500000}).click();
+        cy.get('#channel_display_modeEdit').click();
+        cy.get('#settingTitle').should('contain', 'Channel Display');
+        cy.get('input#channel_display_modeFormatB').click();
+        cy.get('input#channel_display_modeFormatB').next().should('contain', 'Fixed width, centered');
+        cy.get('#saveSetting').click();
+        cy.get('#accountSettingsHeader > .close').click();
```

Comment:
- `500000` is huge timeout. Also you may use constants at `/fixtures/timeouts`.
- Since it shows account settings modal, queries to each element is better (scoped) within the modal -- should use `.within`.
- We're now using `testing-library` as much as possible. So, please use [api queries](https://testing-library.com/docs/dom-testing-library/api-queries) whenever possible. So that queries is represented on how the actual user sees is and not the underlying element IDs, CSS class, etc.

Something like:
```javascript
cy.get('#accountSettingsModal').should('be.visible').within(() => {
    cy.findByText('Display').click();
    cy.findByText('Channel Display').click();
    // do others ...
});
```

Reply:
:heavy_check_mark: